src/session-pool.ts

```diff
@@ -157,9 +158,8 @@ const DEFAULTS: SessionPoolOptions = {
  * Error to be thrown when attempting to release unknown resources.
  */
 export class ReleaseError extends Error {
-  // tslint:disable-next-line no-any
-  resource: any;
-  constructor(resource) {
+  resource: Any;
+  constructor(resource: Session) {
```

Comment:
This is incorrect, this error will only get thrown if whatever the user is attempting to release is not a session. So the value should mirror the `resource` type